jython/ctc/README.md

```diff
@@ -1,4 +1,4 @@
 The jython/ctc directory contains example and utility scripts for the [JMRI USS CTC](http://jmri.org/JavaDoc/doc/jmri/jmrit/ussctc/package-summary.html) package of tools.
 
-The [jython/ctc/TwoColumnMachine.py](http://jmri.org/jython/ctc/TwoColumnMachine.py) animates a sample two-column CTC machine. To use it, load the [xml/samples/TwoColumnMachine.xml](http://jmri.org/xml/samples/TwoColumnMachine.xml) panel file first, then run the sample script.
+The [jython/ctc/TwoColumnMachine.py](http://jmri.org/jython/ctc/TwoColumnMachine.py) animates a sample two-column CTC machine. To use it, load the [jython/ctc//TwoColumnMachine.xml](http://jmri.org/jython/ctc//TwoColumnMachine.xml) panel file first, then run the sample script.
```

Comment:
Is the two slashes intended here?
`//TwoColumnMachine.xml`

Reply:
Fixed, thanks.